README.md

```diff
@@ -1,24 +1,163 @@
-# In Development - DO NOT USE
-
 # Train-Habitat
 
-Train Plugin for connecting to Habitat Supervisor HTTP Gateway
+`train-habitat` is a Train plugin and is used as a Train Transport to connect to Habitat installations.
 
 ## To Install this as a User
 
 You will need InSpec v2.3 or later.
 
 Simply run:
 
+```
 $ inspec plugin install train-habitat
+```
+
+## Using train-habitat from InSpec
+
+As train-habitat takes potentially many options, it is simplest to list the options in your `~/.inspec/config.json` file, then used the named set of options with `-t`.
+
+For example, if your config file contains:
+
+```
+{
+  "file_version": "1.1",
+  "credentials": {
+    "habitat": {
+      "dev-hab": {
+        "api_url": "http://dev-hab.my-corp.io",
+        "cli_ssh_host": "dev-hab.my-corp.io"
+      },
+      "prod-hab": {
+        "api_url": "https://prod-hab.my-corp.io",
+        "api_auth_token": "opensesame"
+      },
+    }
+  }
+}
+```
+
+Using this configuration, you could execute:
+
+```
+$ inspec exec some-profile -t habitat://dev-hab
+# Or
+$ inspec exec some-profile -t habitat://prod-hab
+```
+
+You may also pass `--config some-file.json` to use a config file at a different location.
+
+See below for the full list of options you may use with a `habitat` credential set in your configuration.
+
+## Using train-habitat from Ruby
+
+The options that may be passed to `Train.create` are listed below.
```

Comment:
Is there a particular section that "listed below" refers to? Would it be possible to use a jump link for that?

Reply:
Perhaps I should remove this one.  Here, 'below' means 'immediately following'.

Reply:
D'oh. I finally noticed the header format for this section. I believe it would be okay to leave this sentence as is because the referenced content is immediately afterwards. I thought it was referencing content elsewhere in the document as with my previous comment about jump links.